app/src/main/java/com/github/h3lp3rs/h3lp/MedicalInformation.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.h3lp3rs.h3lp
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import java.time.Year
+
+@RequiresApi(Build.VERSION_CODES.O)
+class MedicalInformation(
+    val size: Int,val weight: Int,val gender: Gender, val yearOfBirth: Int, val conditions: String, val actualTreatment: String, val allergy: String, val bloodType: BloodType,
+){
+    init {
+        require(size in 30..240)
+        require(weight in 30..500)
+        require(yearOfBirth in 1900..Year.now().value)
```

Comment:
With the new constants, you can directly use them here and we avoid the problem of inconsistency if one day we decide to change them